docs/fsharp/tutorials/type-providers/accessing-a-sql-database-entities.md

```diff
@@ -183,17 +183,17 @@ let fullContext = context.DataContext
 let nullable value = new System.Nullable<_>(value)
 
 let addInstructor(lastName, firstName, hireDate, office) =
-let hireDate = DateTime.Parse(hireDate)
-let newPerson = new EntityConnection.ServiceTypes.Person(LastName = lastName,
+  let hireDate = DateTime.Parse(hireDate)
+  let newPerson = new EntityConnection.ServiceTypes.Person(LastName = lastName,
                                                          FirstName = firstName,
                                                          HireDate = nullable hireDate)
```

Comment:
I agree with you @svick. I made the change in my fork. Please advise on what I need to do next. This is my first attempt at a pull request.